packages/haul-babel-preset-react-native/src/transforms/superMemberArrowFunction.ts

```diff
@@ -0,0 +1,72 @@
+import createTemplate from '@babel/template';
+import * as babel from '@babel/core';
+
+/**
+ * With Webpack and babel-loader the following code:
+ * ```js
+ * class Derived extends Super {
+ *   derivedFn1 = async () => {
+ *     await super.superFn1();
+ *   }
+ *
+ *   derivedFn2 = () => {
+ *     super.superFn2();
+ *   }
+ * }
+ * ```
+ * will fail with an exception: `super` cannot be used outside of a class method.
+ * However this is a valid syntax, which successfully run for example on V8 or Chrome,
+ * so we need to transform `super` to use `Super.prototype`:
+ * ```js
+ * class Derived extends Super {
+ *   derivedFn1 = async () => {
+ *     await Super.prototype.superFn1.call(this);
+ *   }
+ *
+ *   derivedFn2 = () => {
+ *     Super.prototype.superFn2.call(this);
+ *   }
+ * }
+ * ```
+ */
+
+const template = createTemplate('SUPER.prototype.METHOD.call(this, ARGS)');
+
+export default function superMemberArrowFunction() {
+  return {
+    visitor: {
+      Program(programPath: babel.NodePath<babel.types.Program>) {
+        programPath.traverse({
+          Super(path) {
+            let parentFunctionPath = path.getFunctionParent();
+
+            // Check if parent function is a async arrow function as a class property.
```

Comment:
Should be `an async` but TBH are you really mean `async`?

Reply:
`an async` needs to be removed